ImageFilterEditor/ImageFilterEditor/Filtering/ApplyFilter/SliderControlsViewController.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  SliderControlsViewController.swift
+//  ImageFilterEditor
+//
+//  Created by Shawn Gee on 5/4/20.
+//  Copyright © 2020 Swift Student. All rights reserved.
+//
+
+import UIKit
+
+protocol SliderControlsDelegate {
+    func sliderAt(index: Int, didChangeValueTo value: Double)
```

Comment:
(This is a *major* nitpick 😅) I'm not sure about this method name; the `at` in the name seems to me to be attached more to the index, and feels weird in the "title" part of the method signature, whereas the `value` seems to me at least to be more important. If it were me I think I'd prefer a title like `sliderValue(atIndex index: Int, didChangeTo newValue: Double)`. Then when referring to the method in documentation, the "shortened" name or the beginning of the name feels more descriptive (`sliderValue` vs `sliderAt`). That's just me and my external eyes looking in, at least.